### PR TITLE
Added yamlloader

### PR DESCRIPTION
**Pull request by rasmusagren**

### CODE REVIEW

**Comment by rasmusagren** [comment]
@bgruening I think this can be merged. Thanks!

File: recipes/yamlloader/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - pyyaml
```
Comment (synapticarbors) [inline recipes/yamlloader/meta.yaml]: I don't think that `pyyaml` is required to `build` the recipe

File: recipes/yamlloader/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+test:
+  imports:
+    - yamlloader
+
+about:
+  home: https://github.com/Phynix/yamlloader
+  license: MIT
+  license_family: MIT
```
Comment (synapticarbors) [inline recipes/yamlloader/meta.yaml]: Please add `license_file: LICENSE`

File: recipes/yamlloader/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  summary: 'Loaders and dumpers for PyYAML'
+
+  description: |
+    This module provides loaders and dumpers for PyYAML. Currently, an OrderedDict
+    loader/dumper is implemented, allowing to keep items order when loading resp.
+    dumping a file from/to an OrderedDict.
+  dev_url: https://github.com/Phynix/yamlloader
+
+extra:
+  recipe-maintainers:
```
Comment (synapticarbors) [inline recipes/yamlloader/meta.yaml]: Please confirm that all of the listed co-maintainers have agreed to be listed.

Comment (rasmusagren) [reply recipes/yamlloader/meta.yaml]: Before you typically asked for the conda-forge people to be added (hmm, or am I thinking of bioconda now?). I haven't asked anyone other than mayou36, who is the author of the package. Should I remove the other ones?

Comment (rasmusagren) [reply recipes/yamlloader/meta.yaml]: @mayou36 Please confirm :)

Comment (mayou36) [reply recipes/yamlloader/meta.yaml]: Hey, thanks for adding it. I agree to be a maintainer

File: recipes/yamlloader/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+#{% set name = "yamlloader" %}
```
Comment (jakirkham) [inline recipes/yamlloader/meta.yaml]: Looks like this `#` snuck in. Fixing with PR ( https://github.com/conda-forge/staged-recipes/pull/5870 ).

Comment (rasmusagren) [reply recipes/yamlloader/meta.yaml]: Oops, don't know how that happened. Thanks!

**Pull request merged**